### ISSUE TITLE
Bug introduced into last version 36, due to python version requirements

### ISSUE THREAD

**Issue by CastagnaIT** [xbmc/addon-check#286]
from  PR #283 merged on v36
has been introduced the use of walrus operator `:=`
https://github.com/xbmc/addon-check/blob/1c9ce9e23d051347388302f1d021d52f3697852b/kodi_addon_checker/check_addon.py#L101-L102

but this is available on python 3.8 and above
the problem is that addon-check can be run also on older python versions

so this raise following error and impossibility to execute the utility
```
Traceback (most recent call last):
  File "/usr/local/bin/kodi-addon-checker", line 5, in <module>
    from kodi_addon_checker.__main__ import main
  File "/usr/local/lib/python3.7/site-packages/kodi_addon_checker/__main__.py", line 14, in <module>
    from kodi_addon_checker import __version__, check_addon, ValidKodiVersions
  File "/usr/local/lib/python3.7/site-packages/kodi_addon_checker/check_addon.py", line 101
    if (extension := parsed_xml.find('extension')) and extension.get('point') == 'kodi.resource.language':
                  ^
SyntaxError: invalid syntax
```

so you need to accomplish same code change in a different way,
or request a higher python version as requirement (that must be available on the Github action)

**Comment by razzeee** [comment]
Should be fixed by https://github.com/xbmc/addon-check/pull/285
